extra/alerts/pcmk_snmp_helper.sh

```diff
@@ -138,7 +138,7 @@ case "$CRM_alert_kind" in
         Cancelled) ;;
         *)
             if [ "${trap_monitor_success}" = "false" ]; then
-                if [[ ${CRM_alert_rc} -eq 0 && "${CRM_alert_task}" == "monitor" ]]; then
+                if [[ ${CRM_alert_rc} -eq 0 || ${CRM_alert_rc} -eq 8 ]] && [ "${CRM_alert_task}" == "monitor" ]; then
```

Comment:
> In POSIX sh, [[ ]] is not supported

should rather be
```
if [ "${CRM_alert_rc}" = 0 ] || [ "${CRM_alert_rc}" = 8 ] \
    && [ "${CRM_alert_task}" = "monitor" ]; then
```
- guard for cases `CRM_alert_rc ` is not defined/non-numeric
- uses `=` rather than `==`
- keep lines at least under 100 characters